phoenicis-scripts/src/main/java/org/phoenicis/scripts/nashorn/NashornScriptInterpreter.java

```diff
@@ -32,7 +32,9 @@ public NashornScriptInterpreter(NashornEngineFactory nashornEngineFactory) {
 
     @Override
     public void runScript(String scriptContent, Consumer<Exception> errorCallback) {
-        nashornEngineFactory.createEngine().eval(scriptContent, errorCallback);
+        final NashornEngine engine = nashornEngineFactory.createEngine();
```

Comment:
Yep. Done. One question: why is the `NashornEngine` created for every call? Can't we just do this once?